src/Umbraco.Web.UI.Client/src/views/propertyeditors/rte/rte.prevalues.html

```diff
@@ -2,35 +2,41 @@
 
 	<umb-control-group label="Toolbar" hide-label="false">
 		<div ng-repeat="cmd in tinyMceConfig.commands">
-			<input type="checkbox" 
-				ng-checked="selected(cmd, cmd.frontEndCommand, model.value.toolbar)" 
-				ng-model="cmd.selected" 
-				ng-change="selectCommand(cmd)"
-				/> 
+            <label>
+                <input type="checkbox"
+                       ng-checked="selected(cmd, cmd.frontEndCommand, model.value.toolbar)"
+                       ng-model="cmd.selected"
+                       ng-change="selectCommand(cmd)" />
 
-				<img ng-src="{{cmd.icon}}" />
-
-				{{cmd.frontEndCommand}}
+                <i class="mce-ico mce-i-{{cmd.frontEndCommand}}"></i>
+                <!--<img ng-src="{{cmd.icon}}" />-->
+                {{cmd.name}}
```

Comment:
Similar ```name``` property like for css.
See also comment for the controller about the font icon.